### PR TITLE
Add dynamicforward option

### PR DESCRIPTION
**Pull request by masa-orca**
##### SUMMARY

Add dynamicforward option to ssh_config module.

##### ISSUE TYPE

- Feature Pull Request
##### COMPONENT NAME

ssh_config
##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

File: changelogs/fragments/ssh_config_add_dynamicforward_option.yml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+minor_changes:
+  - ssh_config - Add dynamicforward option
```
Comment (felixfontein) [inline changelogs/fragments/ssh_config_add_dynamicforward_option.yml:2]:
```suggestion
  - ssh_config - add ``dynamicforward`` option (https://github.com/ansible-collections/community.general/pull/9192).
```

**Review by felixfontein — COMMENTED** [review]
Thanks for your contribution!

**Review by russoz — APPROVED** [review]
LGTM

**Comment by felixfontein** [comment]
@masa-orca thanks for your contribution!
@russoz thanks for reviewing!

**Pull request merged**